docs/hugo/content/guide/metrics.md

```diff
@@ -29,8 +31,75 @@ By default, metrics for ASOv2 are turned on and can be toggled by the following
       containers:
        - args:
          - --metrics-addr=0.0.0.0:8080 (default)    
+         - --secure-metrics=true/false (default: true)
+         - --profiling-metrics=true/false (default: false)
     ```
-   
+
+## Scraping Metrics Securely via HTTPs using RBAC
+
+A ServiceAccount token is required to scrape metrics securely. The corresponding ServiceAccount needs permissions on the "/metrics" and "debug/pprof" paths. 
+This can be achieved e.g. by following the [Kubernetes documentation](https://kubernetes.io/docs/concepts/cluster-administration/system-metrics/).
+
+Follow the steps below to scrape metrics securely.
+
+### ASOv2 Helm Chart
```

Comment:
very minor: Could use one of the table-selector thingies here, like we do for Windows versus Linux in the installation instructions?